educe/stac/rfc.py

```diff
@@ -52,24 +52,30 @@ class BasicRfc(object):
     '''
     def __init__(self, graph):
         self._graph = graph
+        self._nodes = graph.first_outermost_dus()
+        self._points = self._frontier_points(self._nodes)
 
-    def _build_right_frontier(self, points, last):
+    def _build_frontier(self, last):
+        return self._build_frontier_from(last)
+
+    def _build_frontier_from(self, origin):
         """
         Given a dictionary mapping each node to its closest
-        right frontier node, generate a path up that frontier.
+        right frontier nodes and a start node, generate a path
+        up that frontier.
         """
         seen = set()
```

Comment:
If you can declare here `points = self._points`, then the code in the loop can be reverted to what it was, which is slightly more readable, plus